lib/services/registered_addons.pm

```diff
@@ -84,7 +86,8 @@ sub check_suseconnect_cmd {
     diag "$ls_out";
     my $status_out = script_output("SUSEConnect --status-text", 120);
     diag "$status_out";
-    for (my $i = 1; $i < @addons; $i = $i + 1) {
+    for (my $i = 0; $i < @addons; $i = $i + 1) {
+        next if (($addons[$i] eq uc get_var('SLE_PRODUCT')) || ($addons[$i] eq 'SLE_HPC'));
```

Comment:
why skip on above condition?

Reply:
For SUSEConnect --list-extensions it doesn't include product name, so exclude product name and compare modules' name. Before I only exclude the first element, some cases product's name is at the second index.